backend/api/models.py

```diff
@@ -110,3 +110,11 @@ class Session(models.Model):
     time = models.TimeField()
     coach = models.ForeignKey(CustomUser, on_delete=models.CASCADE, related_name='coach_group')
     school = models.ForeignKey(School, on_delete=models.CASCADE, related_name='school_group')
+
+class SwimMeet(models.Model):
+    name = models.CharField(max_length=255, null=False, blank=False)
+    date = models.DateField(null=True, blank=True)
```

Comment:
The user story and the description indicate that this field is required therefore null and blank attributes should be false.